stack/building-pipelines-lsst-io-locally.rst

```diff
@@ -52,43 +52,25 @@ Install Documenteer, the documentation tooling
 ==============================================
 
 Documenteer_ provides tooling to build `pipelines.lsst.io`_.
-Since it’s a `PyPI-distributed Python package <https://pypi.org/project/documenteer/>`__, you need to install it separately from the EUPS Stack.
-The best way to do this is in a Python virtual environment that’s layered on top of the EUPS Stack’s ``site-packages``.
-This way it’s easy to delete Documenteer_ and its dependencies without affecting the Python packages that come with the EUPS Stack.
-
-In the base working directory (that also contains the :file:`pipelines_lsst_io` clone), create the virtual environment and :command:`pip`-install Documenteer_ through the :file:`requirements.txt` file for `pipelines_lsst_io`_:
+Since it’s a `PyPI-distributed Python package <https://pypi.org/project/documenteer/>`__, you need to install it separately from the EUPS Stack:
 
 .. code-block:: bash
 
-   python -m venv --system-site-packages --without-pip pyvenv
-   source pyvenv/bin/activate
-   curl https://bootstrap.pypa.io/get-pip.py | python
-   pyvenv/bin/pip install -r pipelines_lsst_io/requirements.txt
+   cd pipelinies_lsst_io
+   pip install -r requirements.txt
+
+Do this *after* setting up the EUPS Stack.
```

Comment:
(I realise the terminology isn't new here, but...) maybe avoid using “setting up”? It's not immediately obvious whether it means “installing” or running `setup <package>`.